crates/turborepo-lib/src/config.rs

```diff
@@ -296,6 +299,16 @@ impl ConfigurationOptions {
     pub fn env_mode(&self) -> EnvMode {
         self.env_mode.unwrap_or_default()
     }
+
+    pub fn cache_dir(&self) -> &Utf8Path {
+        self.cache_dir.as_deref().unwrap_or_else(|| {
+            Utf8Path::new(if cfg!(windows) {
+                ".turbo\\cache"
+            } else {
+                ".turbo/cache"
+            })
+        })
```

Comment:
nit: this is slightly more idiomatic
```suggestion
        self.cache_dir.as_deref().unwrap_or_else(|| {
            Utf8Path::from_iter([".turbo", "cache"])
        })
```

Reply:
love this kind of feedback! thanks!

@chris-olszewski and I were looking at that and the thinking was that `from_iter` causes an allocation and this doesn't but looking at it again, this does look better.  we were already sorta 50/50 on it anyway :)

Reply:
update: there's no `Utf8Path::from_iter` [from what I can tell](https://docs.rs/camino/latest/camino/struct.Utf8Path.html).

what I was talking about was `join_components`.  would you like to switch to that?